banzai_nres/extract.py

```diff
@@ -32,7 +32,9 @@ def do_stage(self, image: EchelleSpectralCCDData):
             # get the average wavelength: Sum wavelengths weighted by 1 over the vertical width of the trace (e.g. 1/10)
             wavelength[i, x_extent] = self.extract_order(image.wavelengths[this_trace], weights=1/image.wavelengths[this_trace].shape[0])
 
-        image.spectrum = Table({'id': trace_ids, 'wavelength': wavelength, 'flux': flux, 'uncertainty': np.sqrt(variance)})
+        image.spectrum = Table({'id': trace_ids, 'fiber': image.fibers, 'wavelength': wavelength,
```

Comment:
Comment labelled 1 (referenced in a later comment): if image.fibers is a table, this may fail.